lib/data/helpers/provinces.ts

```diff
@@ -30,10 +30,3 @@ export const getContactsPaths = (provinces: Provinces): ContactPath[] => {
   });
   return contactsPaths;
 };
-
-export const getProvinceKebutuhanContactsCount = (
-  contacts: Contact[],
-  kebutuhan: string,
-): number => {
-  return contacts.filter((contact) => contact.kebutuhan === kebutuhan).length;
-};
```

Comment:
Why did you delete this function?

I think you made some mistakes in merging the `main` branch to your branch.
Please fix them, thanks!

Reply:
I didn't. I was thinking the same thing after reading your [comment](https://github.com/kawalcovid19/wargabantuwarga.com/pull/722#issuecomment-899069282) yesterday. Gonna check this